virtool/jobs/db.py

```diff
@@ -186,7 +186,7 @@ async def create_job(
         "workflow": workflow,
         "args": job_args,
         "key": None,
-        "rights": rights.as_dict(),
+        "rights": rights,
```

Comment:
Remove argument. It is never going to be a non-empty dict.
```suggestion
        "rights": {},
```